### ISSUE TITLE
predict_tau()

### ISSUE THREAD

**Issue by yhoriuchi** [yhoriuchi/projoint#38]
Hello, I am currently trying to get the IRR from a conjoint without any repetion, i.e. using the extrapolation method. This seems to work really good, thank you! 
I wonder, is it possible to see based on how many datapoints the different points in the "predict_tau()" functions plot are created? I am just wondering as obviously some of the confidence intervals are quite large. From the code, it seems like this data is available just not stored which is why I thought maybe I could suggest it for another update in the future. 
Again, thanks so much, cool work!

_Originally posted by @brueckmann in https://github.com/yhoriuchi/projoint/discussions/28#discussioncomment-7919779_

**Comment by yhoriuchi** [comment]
Did you try

summary <[https://rdrr.io/r/base/summary.html>(predicted_irr)](https://rdrr.io/r/base/summary.html%3E(predicted_irr))

explained on the following page?

https://yhoriuchi.github.io/projoint/articles/03-predict.html

Yes, some of the confidence intervals could be really large if the number of observations is small. If you think that the predicted IRR is unlikely, I would suggest you simply add `.irr = 0.75` in the `projoint()` function.

**Comment by brueckmann** [comment]
Yes, I did use summary. However, it just gives me the point estimates of the different values, where the point estimate of 0 is the tau. Thanks! In the example you link  https://yhoriuchi.github.io/projoint/articles/03-predict.html it looks like there is 0 conjoints where only 1 conjoint level is different ( no black point), few with 2 (large CI) and some with 3 and so on. And I was just curious, can I access this information in a systemtic way from the projoint package. Thank you!

**Comment by yhoriuchi** [comment]
HI. You can check my code on the GitHub page. If you notice any errors, please let me know! 

https://github.com/yhoriuchi/projoint/blob/master/R/predict_tau.R
